core/src/main/scala/com/spotify/featran/transformers/IQROutlierRejector.scala

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2017 Spotify AB.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.spotify.featran.transformers
+
+import com.spotify.featran.FeatureRejection
+import com.twitter.algebird.QTreeAggregator
+
+/**
```

Comment:
Can you use javadoc style? See this:
https://github.com/spotify/scio/wiki/Style-Guide#documentation-style